### DISCUSSION TITLE
Decouple sponsorship payment method from main account payment method

### DISCUSSION THREAD

**Discussion by ybakos** [community/community#61370] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have identified a barrier for sponsorship recipients that, if changed, could possibly increase the number of payments made to sponsors.

Given that my personal github account uses a payment method that is a credit card owned by my business,
And that I wish to make a sponsorship payment using my personal funds,
I should be able to make a one-time or recurring payment using a payment method of my choice,
And not be forced to change the payment information of my github account.

If my employer is paying for my github fees by letting me attach their payment information to my account, and I want to make a sponsorship payment, GitHub forces me to change the payment information of my account to make the sponsorship payment. Because I do not want to change my account payment method, I am dis-incentivized in making a sponsorship payment.

<img width="1014" alt="Screen Shot 2023-07-20 at 8 34 38 AM" src="https://github.com/community/community/assets/5502/aba876b9-babb-413e-bd1f-643d6eb20951">

I should be able to make a sponsorship payment without changing the main billing information used for my GitHub account.

I can imagine that there are many folks in the same situation, who stop and walk away upon seeing this UX in the sponsorship workflow.

If I could make sponsorship payments using a personal account, while keeping my main GitHub account billing information intact, I would make more sponsorship payments. 💖💖💖💖

**Comment by anjuan** [comment]
> If I could make sponsorship payments using a personal account, while keeping my main GitHub account billing information intact, I would make more sponsorship payments. 💖💖💖💖

Hi, I'm the Engineering Manager for GitHub Sponsors. I can totally understand how challenging it can be to not have flexibility in the payment methods used for sponsorships, and I like your idea! I'll take the idea of having separate payments methods for GitHub billing accounts and sponsorship payments. 

This may not be a desirable option, but the only way to have a separate form of payment currently would be to pay via invoice. This avoids the credit card processing fee but requires an initial spend of $5,000 USD.
